index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>CSS Card Hover Effects Html & CSS</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
```

Comment:
_Responsiveness has a problem @mujeeb100 , contents are being squashed when you reduce the viewport size._

**You can add meta tag to overwrite default viewport behavior.**

```suggestion
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>CSS Card Hover Effects Html & CSS</title>
    <link rel="stylesheet" type="text/css" href="style.css" />
  </head>
```